app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
```

Comment:
```suggestion
    return {
        "line": error["line_number"],
```

Reply:
Fixed in last commit https://github.com/mate-academy/py-linter-formatter/commit/44e9639b2ee43a5f9315d83443f77f4e5ab1a254